### ISSUE TITLE
re-natal breaks with React 0.15.0

### ISSUE THREAD

**Issue by weltan** [drapanjanas/re-natal#35]
React 0.15 came out today, and removes `cloneWithProps` (which is still used by react-native 0.22.2).

re-natal pulls in React 0.15, but produces an error about cloneWithProps.

react-native@0.22.2 still requires react@^0.14.5 to run (probably because of the breaking changes):

`npm WARN react-native@0.22.2 requires a peer of react@^0.14.5 but none was installed.`

I tried downgrading to react@0.14.5, but re-natal does not allow anything below react@0.15, when you run `re-natal use-figwheel`.

Not sure what the workaround is, until React Native fixes the breaking change in React 0.15 and upgrades.

**Comment by drapanjanas** [comment]
Thanks for reporting! 
A workaround which just worked for me is to downgrade react to 0.14 in package.json
```json
"dependencies": {
    "react": "^0.14.5",
    "react-native": "0.22.2"
  }
```
Then run `re-natal deps`. 
Not sure, but probably compatible React version should be also set in package.json same way like react-native version in `re-natal init` command.

**Comment by drapanjanas** [comment]
This is fixed in RN [0.23.1](https://github.com/facebook/react-native/releases/tag/v0.23.1)

**Comment by weltan** [comment]
Thanks!! Amazing turnaround.
